strr-platform-web/app/pages/auth/login.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+const { t, locale } = useI18n()
+const keycloak = useKeycloak()
+const redirectUrl = useRuntimeConfig().public.baseUrl + locale.value + '/platform/dashboard'
```

Comment:
I updated 1password - can you please update the NUXT_BASE_URL inside `vaults.env` to grab from ...`strr/STRR_PLATFORM_URL`